test_pegasus/dax/__main__.py

```diff
@@ -0,0 +1,53 @@
+import sys
+
+import time
+from Pegasus.DAX3 import ADAG, Job, Link, File
+from t1_processing import T1Processing
+from dwi_processing import DWIProcessing
+from coregistration import Coregistration
+from configuration import Configuration, ConfigKey
+from aseg_generation import AsegGeneration
+from output_conversion import OutputConversion
+from tracts_generation import TractsGeneration
+
+if __name__ == "__main__":
+    if len(sys.argv) != 3:
+        sys.stderr.write("Usage: %s DAXFILE\n" % (sys.argv[0]))
+        sys.exit(1)
+    daxfile = sys.argv[1]
+    patient_file = sys.argv[2]
+
+    dax = ADAG("TVB-PIPELINE")
+    dax.metadata("created", time.ctime())
+
+    config = Configuration(patient_file)
+
+    t1_processing = T1Processing(config.props[ConfigKey.T1_FRMT], config.props[ConfigKey.T2_FLAG],
+                                 config.props[ConfigKey.T2_FRMT], config.props[ConfigKey.FLAIR_FLAG],
+                                 config.props[ConfigKey.FLAIR_FRMT], config.props[ConfigKey.OPENMP_THRDS])
```

Comment:
Not critical, just a question:
Could be better to shorten these by having direct properties on a Configuration object ?
config.has_t2, config.has_flair